packages/shared/src/__tests__/broker.test.ts

```diff
@@ -463,9 +463,7 @@ describe(broker.requestSwapDepositAddress, () => {
       brokerConfig,
       'perseverance',
     );
-    expect(postSpy.mock.calls[0][1][0].params[2]).toEqual(
-      '0x2afba9278e30ccf6a6ceb3a8b6e336b70068f045c666f2e7f4f9cc5f47db8972',
-    );
+    expect(postSpy.mock.calls[0][1][0].params[2]).toEqual(destAddress);
```

Comment:
tested broker api on persa and it supports submitting the encoded prolkadot address (pubkey on event matches the one we submitted previously)